### PR TITLE
[Logs] Added a new page for Log Filters.

### PR DESCRIPTION
**Pull request by angelampcosta**

### CODE REVIEW

File: content/logs/reference/logpush-api-configuration/filters.md [hunk 1]
```diff
@@ -0,0 +1,236 @@
...
+
+- Fields specify properties associated with an event.
+
+- Comparison operators define how values must relate to actual request data for an expression to return true.
+
+- Values represent the data associated with fields. When evaluating a rule, Cloudflare compares these values with the actual data obtained from the request.
+
+Filters can be connected using `AND`, `OR` logical operators.
+
+{{<table-wrap style="width:100%">}}
```
Comment (marciocloudflare) [inline content/logs/reference/logpush-api-configuration/filters.md:19]: The width of this table is weird. Maybe add overview to page type and then wrap all text (minus table) in `{{<content-column>}}`?

File: content/logs/reference/logpush-api-configuration/filters.md [hunk 2]
```diff
@@ -0,0 +1,236 @@
...
+      </td>
+    </tr>
+  </tbody>
+</table>
+{{</table-wrap>}}
+
+The filter field has limits of approximately 30 operators and 1000 bytes. Anything exceeding this value will return an error.
+
+{{<Aside type="note" header="Note">}}
+We do not support filtering on the following data types: objects, array[int], array[object].
```
Comment (marciocloudflare) [inline content/logs/reference/logpush-api-configuration/filters.md]:
```suggestion
Cloudflare does not support filtering on the following data types: objects, array[int], array[object].
```

Comment (marciocloudflare) [reply content/logs/reference/logpush-api-configuration/filters.md]: Not sure if it should be Cloudflare or Logs. Also, do we need to place the data types in between backticks?

File: content/logs/reference/logpush-api-configuration/filters.md [hunk 3]
```diff
@@ -0,0 +1,236 @@
...
+{{</Aside>}}
+
+For the Firewall events dataset, we do not support filtering on the following fields: Kind, MatchIndex, Metadata, OriginatorRayID, RuleID, Source.
```
Comment (marciocloudflare) [inline content/logs/reference/logpush-api-configuration/filters.md]:
```suggestion
For the Firewall events dataset, Cloudflare does not support filtering on the following fields: Kind, MatchIndex, Metadata, OriginatorRayID, RuleID, Source.
```

File: content/logs/reference/logpush-api-configuration/filters.md [hunk 4]
```diff
@@ -0,0 +1,236 @@
...
+
+For the Gateway HTTP dataset, we do not support filtering on the following fields: Downloaded File Names, Uploaded File Names.
```
Comment (marciocloudflare) [inline content/logs/reference/logpush-api-configuration/filters.md]:
```suggestion
For the Gateway HTTP dataset, Cloudflare does not support filtering on the following fields: Downloaded File Names, Uploaded File Names.
```

**Review by marciocloudflare — APPROVED** [review]

File: content/logs/reference/logpush-api-configuration/filters.md [hunk 5]
```diff
@@ -0,0 +1,254 @@
...
+
+For the Firewall events dataset, the following fields are not supported: Kind, MatchIndex, Metadata, OriginatorRayID, RuleID, Source.
+
+For the Gateway HTTP dataset, the following fields are not supported: Downloaded File Names, Uploaded File Names.
+
+Example request using cURL:
+
+```bash
+curl -s -X POST https://api.cloudflare.com/client/v4/zones/<ZONE_ID>/logpush/jobs/ \
+-H 'X-Auth-Key: <KEY> \
```
Comment (tanushree-sharma) [inline content/logs/reference/logpush-api-configuration/filters.md]: missing an end quote here

File: content/logs/reference/logpush-api-configuration/filters.md [hunk 6]
```diff
@@ -0,0 +1,254 @@
...
+-H 'X-Auth-Email:<EMAIL>' \
+-H 'Content-Type: application/json' \
+-d '{
+"name":"Burritobot static assets"
```
Comment (tanushree-sharma) [inline content/logs/reference/logpush-api-configuration/filters.md]: missing a comma at the end of this line

File: content/logs/reference/logpush-api-configuration/filters.md [hunk 7]
```diff
@@ -0,0 +1,254 @@
...
+pcx-content-type: how-to
+type: overview
+title: Filters
+weight: 40
+layout: list
+---
+
+# Filters
+
+The following table represents the comparison operators that are supported and example values. Filters are added as escaped JSON strings formatted as `{\"key\":\"<field>\",\"operator\":\"<comparison_operator>\",\"value\":\"<value>\"}`.
```
Comment (tanushree-sharma) [inline content/logs/reference/logpush-api-configuration/filters.md]: can we replace the sample format here to `{"key":"<field>","operator":"<comparison_operator>","value":"<value>"}` instead

**Review by tanushree-sharma — APPROVED** [review]

**Pull request merged**